crates/net/eth-wire/src/errors/eth.rs

```diff
@@ -10,9 +10,13 @@ pub enum EthStreamError {
     #[error(transparent)]
     P2PStreamError(#[from] P2PStreamError),
     #[error(transparent)]
+    ParseVersionError(#[from] ParseVersionError),
+    #[error(transparent)]
     EthHandshakeError(#[from] EthHandshakeError),
     #[error("message size ({0}) exceeds max length (10MB)")]
     MessageTooBig(usize),
+    #[error("TransactionHashes invalid len of fields: {0} {1} {2}")]
+    TransactionHashesInvalidLenOfFields(usize, usize, usize),
```

Comment:
I'd like to use named fields here, otherwise it's harder to understand what the different usizes are

Reply:
follow it up with 231f46085521ada2ea6697bdd8f83baf971faa35.